mod/racing_plus/struct.lua

```diff
@@ -0,0 +1,210 @@
+--[[
```

Comment:
can we add a comment header of the URL of where you got the file from for reference?

Reply:
what do you mean add the URL it is literally one line below? =)